charts/sysdig-deploy/README.md

```diff
@@ -51,7 +51,7 @@ Currently included components:
        --set global.clusterConfig.name=CLUSTER_NAME
     ```
 
-     **GKE Autopilot**: GKE Autopolot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install Sysdig agent:
+     **GKE Autopilot**: GKE Autopilot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install Sysdig agent:
```

Comment:
**GKE Autopilot**: GKE Autopilot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install the Sysdig agent:

Reply:
I think the suggested change looks the same as the original one

Reply:
I only noticed it after pasting, but the new text says "install **the** Sysdig agent" which sounds more right

Reply:
ah my bad!